preview/mail-processing.html

```diff
@@ -0,0 +1,320 @@
+  <!-- Header -->
+    <header id="header" class="alt">
+      <div class="logo"><a href="http://127.0.0.1:4000/index.html" alt="Apache James"><img src="mail-processing_files/james.svg" alt="james logo"></a></div>
+      <h1 class="hidden">James Enterprise Mail Server</h1>
+      <h2>Emails at the heart of your business logic</h2>
+    </header>
+
+  <!-- Main -->
+    <div id="main">
+
+      <!-- Introduction -->
+        <section id="intro" class="main special">
+          <div class="">
+            <div class="content align-left">
+              <header class="major">
+                <h1>How to customize mail processing...</h1>
+              </header>
+                <header class="major">
+                    <h2><b>Mail processing component overview</b></h2>
+                </header>
+              <p class="align-left">At the heart of James lies the Mailet container, which allows mail processing. This is
+              split in smaller units, with specific responsibilities:
+
+              </p>
+
+              <ul class="no-padding">
+                <li><b>Mailets:</b> Are operations performed with the mail: modifying it, performing a side-effect, etc...</li>
+                <li><b>Matchers:</b> Are per-recipient conditions for mailet execution</li>
+                <li><b>Processors:</b> Are matcher/mailet pair execution threads</li>
+              </ul>
+
+              <p> Read <a href="http://127.0.0.1:4000/server/feature-mailetcontainer.html">this</a> for more explanations of mailet container concepts.</p>
+
+              <p>One can define the mailet container content through the <a href="http://127.0.0.1:4000/server/config-mailetcontainer.html">mailetcontailer.xml</a> file.
+              Hence, we can arrange James standard components listed <a href="http://127.0.0.1:4000/server/dev-provided-mailets.html">here</a> to achieve basic logic. But what if our goal are more
+              complex? What if we need our own processing components?</p>
+
+              <p>This page will propose a 'hands on practice' how-to using James 3.0.1. We will implement a custom mailet and a custom matcher,
+              then deploy it in a James server.</p>
+
+              <p>We need to choose our use case. We will, when a mail is older than a day, write a mail to the original sender
+              to inform him about the delay, say that we are sorry, and send him a promotion code...</p><pre></pre><p></p>
+
+
+              <header class="major">
+                <h2><b>Writing custom mailets and matchers</b></h2>
+              </header>
+
+              <p>None of the matchers and mailets available in James allows us to implement what we want. We will have to
+              write our own mailet and matcher in a separated maven project depending on James Mailet API.</p>
+
+              <p>We will write a <b>IsDelayedForMoreThan</b> mathcer with a configurable delay. If the Sent Date of incoming emails is older than specified delay, then we
```

Comment:
`older than specified delay` -> `older than a specified delay`?
`then we should match` -> `then emails should be matched`